clients/src/test/java/org/apache/kafka/common/network/Tls13SelectorTest.java

```diff
@@ -46,6 +48,20 @@ protected Map<String, Object> createSslClientConfigs(File trustStoreFile) throws
         return configs;
     }
 
+    @Flaky(value = "KAFKA-14249", comment = "Copied from base class. Remove this override once the flakiness has been resolved.")
+    @Test
+    @Override
+    public void testCloseOldestConnection() throws Exception {
+        String id = "0";
```

Comment:
How about `super.testCloseOldestConnection()`?

Reply:
Can you tell I don't use inheritance often? 🙃